README.md

```diff
@@ -30,6 +30,7 @@ Know a resource that isn't listed below? Feel free to create a new [pull request
 | [Co-op Design Manual](https://coop-design-manual.herokuapp.com/) | 👍 | 👍 | 👍 |  |
 | [Dropbox Scooter](http://dropbox.github.io/scooter/) | 👍 |  |  |  |
 | [Duolingo Design Guidelines](https://www.duolingo.com/design) | 👍 |  | 👍 |  |
+| [Firefox Photon Design System](http://design.firefox.com/photon/) |  | 👍 |  |  |
```

Comment:
It's already in the list as "Mozilla Photon Design System" ;)

Reply:
Oh, really, didn't notice it!